runtime/util/cphelp.c

```diff
@@ -153,14 +153,14 @@ getModuleJRTURL(J9VMThread *currentThread, J9ClassLoader *classLoader, J9Module
 			}
 		} else {
 			/* its java.base module */
-#define JRT_JAVA_BASE_URL "jrt:/java.base"
-			jrtURL = j9mem_allocate_memory(sizeof(jrtURL->length) + sizeof(JRT_JAVA_BASE_URL) - 1, OMRMEM_CATEGORY_VM);
+			J9_DECLARE_CONSTANT_UTF8(JRT_JAVA_BASE_URL, "jrt:/java.base");
```

Comment:
Since JRT_JAVA_BASE_URL is no longer a #define, should it sill be all caps?

Reply:
Good point - renamed to `jrtJavaBaseUrl`